README.md

```diff
@@ -109,6 +109,32 @@ require("gitlab").setup({
     resolved = '✓', -- Symbol to show next to resolved discussions
     unresolved = '✖', -- Symbol to show next to unresolved discussions
   },
+  discussion_sign = {
+    -- See :h sign_define for details about sign configuration.
+    enabled = true,
+    text = "💬",
+    linehl = nil,
```

Comment:
I see, I will try to add also callback to remove highlights and signs when reviewer is closed or left ( will see what options there are available ).

Regarding the `linehl` you are right that highlight is applied only on unchanged lines ( if there is comment ) - I need to check if it is possible to define some highligh priority or something like that. The idea here was to make all settings available to user but I didn't try all of them. If nothing more it might be at least useful to see context of comment outside of changed lines :thinking:

Reply:
The problem with signs and diagnostics outside of reviewer is fixed.